WORKSPACE

```diff
@@ -35,8 +35,8 @@ bind(
 # When updating envoy sha manually please update the sha in istio.deps file also
 #
 # Determine SHA256 `wget https://github.com/envoyproxy/envoy/archive/COMMIT.tar.gz && sha256sum COMMIT.tar.gz`
-ENVOY_SHA = "b3be5713f2100ab5c40316e73ce34581245bd26a"
-ENVOY_SHA256 = "79629284ae143d66b873c08883dc6382fac2e8ed45f6f3521f7e7282b6650216"
+ENVOY_SHA = "925810d00b0d3095a8e67fd4e04e0f597ed188bb"
```

Comment:
This does look a bit risky for release branch, have you take a look on changes (particularly on version_history and DEPRECATED.md)?

Reply:
1.1 isn't released yet, and the proxy has been constantly updated (most recently 2 weeks ago).

Reply:
yes I assume it is very close to be released? The old SHA is 3 weeks ago.

Reply:
The old SHA is from 1/24 (so 2 weeks and 1 day ago), and it doesn't look that there were any additions to the `DEPRECATED.md` since then.

While we could cherry-pick a few fixes that are needed, I'm not sure if this is such a great idea.

Also, no release candidate has been cut yet, so I'm not sure if "very close" is very accurate.

Reply:
We are trying to cut the first rc now, but this is a critical change.  I think we have to take the risk